### DISCUSSION TITLE
WT Config

### DISCUSSION THREAD

**Discussion by Yanks1023** [gpstar81/GPStar-proton-pack#78] [General]
So I believe I was able to update the firmware on both of my wav trig boards. How do you confirm this on the Wtconfig software and is there anything I need to do in that config software (create Init file?)

**Comment by gpstar81** [comment]
I believe the software updater tells you when it was successful in sending over the firmware you selected.

Nothing is required in the config software

**Reply by Yanks1023** [reply]
Yes, I got the success message, but didn't see anything other than that to confirm it. Thank you!
